app/backend/src/clj/teet/map/map_services.clj

```diff
@@ -112,12 +112,26 @@
        (handle-wfs-response wfs-url (:TYPENAME query-params)
                             request-delay (or custom-parse-feature parse-feature))))))
 
-(defn wfs-get-feature [{:keys [wfs-url]} typename ogc-filter]
+(defn- return-properties->string
+  "Transforms a vec of keyword property names into a comma separated list:
+   [:a :b :c] -> \"a,b,c\""
+  [return-properties]
+  (->> return-properties
+       (map name)
+       (str/join ",")))
+
+(defn wfs-get-feature
+  "Returns features of type `typename`, filtered by `ogc-filter`."
+  [{:keys [wfs-url]}
+   typename                                     ;; The `typeName` of the feature
+   ogc-filter                                   ;; the OGC filter in hiccup format
+   & [{:keys [return-properties] :as _options-map}]] ;; a map of other options; `return-properties` is a vector of property keywords to return
```

Comment:
should this comment instead be in the docstring as it describes input parameters